bundler.d/journald.rb

```diff
@@ -0,0 +1,6 @@
+# Disable to avoid journald native gem in development setup.
```

Comment:
shouldn't this be `bundler.plugins.d` (or the other file there also be in `bundler.d`)?

Reply:
nvm